infra/build/Makefile

```diff
@@ -92,26 +93,9 @@ build-image-developer-tools:
 		--build-arg TINKEY_VERSION=${TINKEY_VERSION} \
 		--build-arg ALPINE_VERSION=${ALPINE_VERSION} \
 		--build-arg MODULE_SWAPPER_VERSION=${MODULE_SWAPPER_VERSION} \
+		--build-arg TFLINT_BP_PLUGIN=${TFLINT_BP_PLUGIN} \
 		-t ${DOCKER_IMAGE_DEVELOPER_TOOLS}:${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} "./developer-tools"
-	docker build \
```

Comment:
Probably want to pull the later tag/push for DOCKER_IMAGE_DEVELOPER_TOOLS_JENKINS and DOCKER_IMAGE_DEVELOPER_LIGHT?

Reply:
not removing, reverted these bits